web/demos/drupal-multisite/index.html

```diff
@@ -0,0 +1,164 @@
+
+  <!-- Asciinema -->
+  <link rel="stylesheet" type="text/css" href="../../asciinema/asciinema-player.css" />
+  <script src="../../asciinema/asciinema-player.js"></script>
+
+  <!-- Custom Platform.sh scripts -->
+  <script src="../../asciinema-start.js"></script>
+
+  <link rel="apple-touch-icon" sizes="57x57" href="../../favicon/apple-icon-57x57.png">
+  <link rel="apple-touch-icon" sizes="60x60" href="../../favicon/apple-icon-60x60.png">
+  <link rel="apple-touch-icon" sizes="72x72" href="../../favicon/apple-icon-72x72.png">
+  <link rel="apple-touch-icon" sizes="76x76" href="../../favicon/apple-icon-76x76.png">
+  <link rel="apple-touch-icon" sizes="114x114" href="../../favicon/apple-icon-114x114.png">
+  <link rel="apple-touch-icon" sizes="120x120" href="../../favicon/apple-icon-120x120.png">
+  <link rel="apple-touch-icon" sizes="144x144" href="../../favicon/apple-icon-144x144.png">
+  <link rel="apple-touch-icon" sizes="152x152" href="../../favicon/apple-icon-152x152.png">
+  <link rel="apple-touch-icon" sizes="180x180" href="../../favicon/apple-icon-180x180.png">
+  <link rel="icon" type="image/png" sizes="192x192"  href="../../favicon/android-icon-192x192.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../../favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="../../favicon/favicon-96x96.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../../favicon/favicon-16x16.png">
+  <link rel="manifest" href="../../favicon/manifest.json">
+  <meta name="msapplication-TileImage" content="/favicon/ms-icon-144x144.png">
+
+  <style type="text/css">
+    /* Put presentation-specific CSS here. Yes, it will happen. */
+  </style>
+
+</head>
+
+<!-- ontouchstart="" will enable :hover and :active events on tap gesture for PWAs -->
+<body ontouchstart="">
+
+<div class="reveal">
+
+  <!-- Any section element inside of this container is displayed as a slide -->
+  <div class="slides">
+
+    <section id="title-slide" data-background-color="#4786ff">
+      <h1>Drupal 8 Multi-site on Platform.sh</h1>
+    </section>
+
+    <section>
+      <blockquote>What about Drupal multi-site?</blockquote>
+    </section>
+
+    <section>
+    <div class="two-col-svg">
+      <div>Setting up a Drupal 8 multi-site app with subdomains takes just a few modifications from our existing template.</div>
+      <div><img src="assets/druplicon.svg" class="plain" width="400px" alt="Composer" /></div>
+    </div>
+    </section>
+
+    <section>
+      <img src="assets/01-routes.png" alt="Multi-site routes." width="75%"/>
+      <p style="margin-top: 2em;">Each site gets a unique subsite ID</p>
+      <p>that is used to define a its subdomain in <code>routes.yaml,</code></p>
+    </section>
+
+    <section>
+      <img src="assets/02-settings.png" alt="Multi-site settings directories." />
+      <p style="margin-top: 2em;">their own <code>config/sync</code> directory as well as settings directories</p>
```

Comment:
Even following the phrase on the previous slide, this doesn't flow. You switched from "each...its" to "their".

Additionally, this sentence is structured like a list but lacks a conjunction.